Formula/ospac-gui.rb

```diff
@@ -0,0 +1,20 @@
+class OspacGui < Formula
+  desc "Open Source Podcast Audio Chain - GUI"
+  homepage "http://ospac.net/"
+  url "https://github.com/sritterbusch/ospac/archive/v1.0.tar.gz"
+  sha256 "793f0454cebe48dfa8364f26fa48469717954c31cb632a9c0b9dd24920d83f4c"
+
+  depends_on "libsndfile"
+  depends_on "fltk"
+  depends_on "ospac"
+
+  def install
+    system "make", "ospac-gui"
```

Comment:
Of course there is "make install" to /usr/local/bin. Since the homebrew build process does not seem to be in a jail, this would not result in a target where home-brew expects it to be. Therefore I added the two install commands directly into the script taking care of the demanded destinations.